XIPs/xip-10-brandinfo-schema.md

```diff
@@ -0,0 +1,90 @@
+---
+xip: 14
+title: Conversation context metadta schema
+description: Schema to enable standardized sharing of conversation context
+author: Yash Lunagaria (@yash-luna)
+status: Review
+type: Standards Track
+category: XRC
+created: 2023-01-31
+---
+
+## Abstract
+
+This XIP proposes a schema for sharing additional context about a conversation beyond a `conversationId`. Having a standard way to provide not only a conversationId, but also conversation metadata will enable different frontends to display useful contextual information in a consistent, user-friendly manner, while still affording apps design and UI flexibility.
+
+## Motivation
+
+The SDK API currently accepts a unique `conversationId` per address pair and allows `metadata` to be set as well. Today, different apps use these parameters in different ways. Therefore an app may not know how to correctly render in its UI `conversationId` and `metadata` for conversations originating from other apps. We propose a schema to standardize the use of conversation `metadata` for the purpose of app brand expression stemming from the `conversationId`. The `metadata` field may include other non-standard (or future standard) properties.
+
+Note that the schema is only applicable when setting a non-null `conversationId`. If `conversationId` is null, the SDK does not allow setting `metadata`.
+
+## Specification
+
+Proposed `displayInfo` schema in conversation `metadata`
+
+```json
+{
+    conversationId: "mydomain.xyz/abc/qrs",
+    metadata: { 
+        "displayInfo.prettyName": "My company", 
+        "displayInfo.profileImage": "mydomain.xyz/assets/myimage.png", 
```

Comment:
How does `displayInfo.profilePicture` differ from a `Contact ProfilePicture` or a `group conversation image`? Is the intention different?

Reply:
This XIP started with the idea of brandInfo/clientMetadata in mind but we realized that the application is broader - i.e. to add context beyond conversationId that allows a frontend to contextualize conversations

Sample UIs and non-clientMetadata examples are work in progress. Sharing most recent UI mock that highlights how they can be utilized for clientMetadata, NftMetadata, or other metadata that contextualizes the conversation.

![image](https://user-images.githubusercontent.com/118013479/217127692-62f0b8e5-87c1-4ea7-b2ae-f7ee320b5c36.png)

Reply:
This example is very helpful. 

I might suggest that we scope the properties to something more descriptive than `DisplayInfo.DisplayName`. In particular we should create specific terms around what to collectively call the OpenSea/Lens tags.

e.g. `ConversationSource.DisplayName`, `ConversationOrigin.DisplayName`, `AppSource....`

My main thought here is that with groupchats coming in the near future, `DisplayName` will become a loaded term.